config/checkstyle/checkstyle.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm not sure if you have this problem, but <gradle build> seems to be failing for me because of trailing whitespace on empty lines. If you also have this issue, then that should either be fixed either in the checkstyle or in the source files.

For reference, my <gradle -v> output:
------------------------------------------------------------
Gradle 1.4
------------------------------------------------------------

Gradle build time: Monday, September 9, 2013 8:44:25 PM UTC
Groovy: 1.8.6
Ant: Apache Ant(TM) version 1.9.3 compiled on April 8 2014
Ivy: non official version
JVM: 1.7.0_75 (Oracle Corporation 24.75-b04)
OS: Linux 3.13.0-45-generic amd64